Content/posts/105-20240610.md

```diff
@@ -0,0 +1,55 @@
+---
+date: 2024-06-10 09:00
+description: Swift 5.10.1リリース、Swift Collections 1.1.1リリース、Nuke 12.7.2リリース、Realm v10.51.0リリース、swift-composable-architecture 1.11.0リリース
+tags: swift, swift-collections, swift-package-manager, nuke, realm, swift-composable-architecture
+---
+
+# 105 2024-06-10
+
+## OSS のリリース情報
+
+iOS アプリ開発でよく使われている OSS のリリース情報です。
+
+### Apple
+
+#### Swift 5.10.1
+
+[https://github.com/apple/swift/releases/tag/swift-5.10.1-RELEASE](https://github.com/apple/swift/releases/tag/swift-5.10.1-RELEASE)
+
+#### Swift Collections 1.1.1
+
+[https://github.com/apple/swift-collections/releases/tag/1.1.1](https://github.com/apple/swift-collections/releases/tag/1.1.1)
+
+- デバッグビルド時の不具合が修正されました。
+- ライブラリ内の不安定なモジュールの名前がリネームされました。
+
+#### Swift Package Manager 5.10.1 Release
+
+[https://github.com/apple/swift-package-manager/releases/tag/swift-5.10.1-RELEASE](https://github.com/apple/swift-package-manager/releases/tag/swift-5.10.1-RELEASE)
+
+- Swift のバージョンが 5.10.1 に上げられました。
```

Comment:
30:32  ✓ error  Should remove period mark("。") at end of list item       period-in-list-item

```suggestion
- Swift のバージョンが 5.10.1 に上げられました
```